samples/config/prod/m-cluster/configstore.yaml

```diff
@@ -1,11 +1,14 @@
 image:
   #repository: forgerock-docker-public.bintray.io/ds
   repository: gcr.io/engineering-devops/ds
-  tag: 6.5.0-M0
+  tag: 6.5.0-M129.2
   pullPolicy: Always
   
 instance: configstore
 
+cts:
```

Comment:
What is this? (For my information, never saw it before. Is it something I need to add to the DevOps Guide?)

Reply:
This has always been there in the original charts values.yaml.  I am only exposing it now in m-cluster.